package.json

```diff
@@ -32,29 +32,30 @@
     "eslint-plugin-node": "^11.0.0",
     "eslint-plugin-promise": "^4.2.1",
     "eslint-plugin-react": "^7.19.0",
-    "eslint-plugin-react-hooks": "^2.5.1",
+    "eslint-plugin-react-hooks": "^4.1.2",
     "eslint-plugin-standard": "^4.0.1",
     "eslint-webpack-plugin": "^2.5.3",
     "gettext-parser": "^2.0.0",
     "htmlparser": "^1.7.7",
     "jed": "^1.1.1",
-    "mini-css-extract-plugin": "^0.11.0",
-    "null-loader": "^3.0.0",
+    "mini-css-extract-plugin": "^1.4.0",
+    "null-loader": "^4.0.0",
     "sass": "^1.35.1",
     "sass-loader": "^12.1.0",
-    "sizzle": "^2.3.3",
-    "stdio": "^0.2.7",
+    "sizzle": "^2.3.5",
+    "stdio": "^2.1.1",
     "string-replace-loader": "^3.0.0",
     "svgo": "1.3.0",
     "terser-webpack-plugin": "^5.1.3",
-    "webpack": "^5.31.0",
-    "webpack-cli": "^4.6.0"
+    "webpack": "^5.38.0",
+    "webpack-cli": "^4.7.0"
   },
   "dependencies": {
-    "@patternfly/patternfly": "4.115.2",
-    "@patternfly/react-core": "4.135.0",
-    "@patternfly/react-styles": "4.11.0",
-    "@patternfly/react-table": "4.29.0",
+    "@patternfly/patternfly": "4.125.3",
+    "@patternfly/react-core": "4.147.0",
+    "@patternfly/react-icons": "4.11.8",
```

Comment:
This is indirect deps of core - no need to specify explicitely.

Reply:
See npm ls @patternfly/react-icons for the deps info.

Reply:
True, it's a bit strange that it's not defined in the `package.json` of cockpit as it imports it directly:
```
src/lib/cockpit-components-plot.jsx:import { AngleLeftIcon, AngleRightIcon, SearchMinusIcon } from '@patternfly/react-icons';
```

Reply:
That's true. Aright, let's explicitly install it then